### ISSUE TITLE
[BUG] AzeritUI Layout Resetting 

### ISSUE THREAD

**Issue by Scruffynerfherding** [GoldpawsStuff/AzeriteUI5#17]
### Is this issue for AzeriteUI for Dragonflight?

- [X] Yes, I understand. This is for Dragonflight.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

It appears that the reset layout bug has returned. Every time I log out and back in on a character, my entire AzeriteUI layout resets. 

### To reproduce

Log out and then log back in. 

### Additional context

None

**Comment by Goldpaw** [comment]
Yes, keeps happening, I'm aware. We have talked about this in the patron section of our discord, and come to the conclusion that the full automation and editmode integration I have attempted just isn't working. 

So I'll be rewriting the editmode integration to have an extra config panel next to the editmode window, where one can load/save azeriteui presets, separate from the editmode ones. One will have to click manually for this, they won't automatically change with the editmode. But the choices will be remembered through reloads and relogs. 

I have no ETA, but it's pretty much my top prio now.
